src/containers/passwordReset/RequestPasswordReset.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import RequestPasswordReset from '../../components/passwordReset/RequestPasswordReset';
+import { requestPasswordResetAction } from '../../store/actions/async/passwordResetActions';
+import { resetState } from '../../store/actions/sync/passwordResetActions';
+import '../../assets/styles/Signup.scss';
+
+export class PasswordRequestContainer extends Component {
+
+    state = {
+        buttonDisabled: true,
+        email: "",
+    }
+    onInputChange = (e) => {
+        const emailRegex = /^([a-zA-Z0-9_\-\.]{1,})@([a-zA-Z0-9_\-\.]{1,})\.([a-zA-Z]{2,5})$/;
+        this.setState(
+            {buttonDisabled: !emailRegex.test(e.target.value),
+                email : e.target.value}
+        )
+    }
+
+    onSubmit = () => {
+        const { requestPasswordReset } = this.props;
+        requestPasswordReset(this.state.email);
+    };
+
+    render() {
+        let errorMsg = null
+        let showError = false
+        let incomingError = this.props.error
+
+        if (incomingError !== null) {
+            errorMsg = (
+                ('user' in incomingError)? incomingError.user.message :
+            incomingError.errors[0])
+            showError = true
+        }
+
+        return (
+            <div className="Signup">
+                <RequestPasswordReset
+                    disabled={this.state.buttonDisabled}
```

Comment:
Make use of destructuring to prevent duplication of `this.state` and `this.props`